index.php

```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html lang="en-CA">
 <head>
+    		<script>
+	function gotoregister() {
+    location.href = "register.php";
```

Comment:
Indentation here. Also we should name the function properly, let's use snake case. So `function goto_register()`